tests/operator/operator.go

```diff
@@ -1985,7 +1969,8 @@ spec:
 				}, 60*time.Second, time.Second).ShouldNot(HaveOccurred())
 
 				By("creating a simple VMI")
-				_, err = virtClient.VirtualMachineInstance(testsuite.GetTestNamespace(nil)).Create(context.Background(), tests.NewRandomVMIWithEphemeralDisk(cd.ContainerDiskFor(cd.ContainerDiskCirros)), metav1.CreateOptions{})
+				vmi := libvmifact.NewCirros()
+				_, err = virtClient.VirtualMachineInstance(testsuite.GetTestNamespace(nil)).Create(context.Background(), vmi, metav1.CreateOptions{})
```

Comment:
May I ask whi don't you update `vmi`? This would make it carry the namespace and various defaults.

Reply:
ah, that's what was here before.